### PR TITLE
Edit create action to send back the article id

### PR DESCRIPTION
**Pull request by raptorf1**
# [PT Story](https://www.pivotaltracker.com/story/show/166271082)

## Changes proposed in this PR
* Test for json returning article id as soon as article is created
* Edit create action in Articles controller to make that happen

## Thing we learned
* Binding.pry rules!

### CODE REVIEW

File: app/controllers/api/v1/articles_controller.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def index
   def create
     article = Article.create(article_params)
     if article.persisted?
-      render json: { message: 'Successfully created' }
+      render json: { message: 'Successfully created', data: article.id }
```
Comment (tochman) [inline app/controllers/api/v1/articles_controller.rb]: I would not use `data here. Why not use `article_id`?

**Review by tochman — CHANGES_REQUESTED** [review]
Please take a close look at `spec/requests/api/v1/articles_spec.rb` you are making 2 calls to the bd after your refactoring. I would probably use yet another describe block...

File: app/controllers/api/v1/articles_controller.rb (hunk 1 above)
Comment (raptorf1) [reply app/controllers/api/v1/articles_controller.rb]: changed

**Comment by raptorf1** [comment]
> Please take a close look at `spec/requests/api/v1/articles_spec.rb` you are making 2 calls to the bd after your refactoring. I would probably use yet another describe block...

Changed also.

**Review by tochman — APPROVED** [review]
Great job!

**Pull request merged**